hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JetBootstrap.java

```diff
@@ -126,7 +126,12 @@ public static synchronized void executeJar(@Nonnull Supplier<JetInstance> suppli
         } finally {
             JetInstance remembered = JetBootstrap.supplier.remembered();
             if (remembered != null) {
-                remembered.shutdown();
+                try {
+                    remembered.shutdown();
+                } catch (Throwable t) {
+                    System.err.println("Client shutdown failed with:");
```

Comment:
Shouldn't we use the `LOGGER` that's in the class?

Reply:
we use the `LOGGER` only in `createStandaloneInstance`, rest of the interaction with the user always happens on `System.err`